doc/release_notes/release_notes_3.0.rst

```diff
@@ -79,3 +144,6 @@ Fixed Issues Details
 Known Issues
 ************
 
+- :acrn-issue:`6631` - [KATA] Kata support is broken in v2.7
+- :acrn-issue:`6978` - [TGL] openstack failed with ACRN v2.7
+- :acrn-issue:`7827` - [Configurator]Pre_launched standard VMs cannot share CPU with Service VM
```

Comment:
I find this one an odd one, since it was always my understanding that it was not possible to share CPUs between pre-launched VMs. Or did I have the wrong understanding?

Reply:
Pre-launched standard VMs should be able to share CPU with service VM.